pkg/controllers/multiclusterservice/mcs_controller.go

```diff
@@ -603,6 +606,22 @@ func (c *MCSController) SetupWithManager(mgr controllerruntime.Manager) error {
 		Complete(c)
 }
 
+func (c *MCSController) serviceHasCrossClusterMultiClusterService(svc *corev1.Service) bool {
+	mcs := &networkingv1alpha1.MultiClusterService{}
+	if err := c.Client.Get(context.Background(),
+		types.NamespacedName{Namespace: svc.Namespace, Name: svc.Name}, mcs); err != nil {
+		if !apierrors.IsNotFound(err) {
+			klog.Errorf("Failed to get MultiClusterService(%s/%s):%v", svc.Namespace, svc.Name, err)
+		}
+		return false
+	}
+
+	if !helper.MultiClusterServiceCrossClusterEnabled(mcs) {
```

Comment:
Can be replaced with 'return helper.MultiClusterServiceCrossClusterEnabled(mcs)'